apps/deploy-web/src/components/providers/ProviderRawData.tsx

```diff
@@ -11,11 +11,19 @@ import Layout from "../layout/Layout";
 import { CustomNextSeo } from "../shared/CustomNextSeo";
 import ProviderDetailLayout, { ProviderDetailTabs } from "./ProviderDetailLayout";
 
+export const COMPONENTS = {
+  Layout,
+  CustomNextSeo,
+  ProviderDetailLayout,
+  DynamicReactJson
+};
+
 type Props = {
   owner: string;
+  components?: typeof COMPONENTS;
```

Comment:
optional property to pass components used in rendering, fallbacks to default value, so in reality this thing will be only used inside unit tests. Potentially can be used in some advanced cases